clientv3/ordering/util_test.go

```diff
@@ -142,8 +142,9 @@ func TestUnresolvableOrderViolation(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	clus.Members[3].WaitStarted(t)
 	cli.SetEndpoints(clus.Members[3].GRPCAddr())
-	time.Sleep(1 * time.Second) // give enough time for operation
+	time.Sleep(5 * time.Second) // give enough time for operation
```

Comment:
This is the `time.Sleep` that we shouldn't need to increase. I'll be looping back on this shortly.

Reply:
Sure, let's follow up in next commits. Thanks